_posts/2020-14-02-creating-doc-assemblies.md

```diff
@@ -0,0 +1,178 @@
+---
+layout: post
+title:  "Creating Strimzi documentation assemblies"
+date: 2020-14-02
```

Comment:
I think the data has format YYYY-MM-DD ... I think this "killed" the build.

Reply:
Happy Valentine's Day. :smile: